src/SFA.DAS.EmployerApprenticeshipsService.Domain/Models/Role.cs

```diff
@@ -10,26 +10,33 @@ public enum Role
         Viewer = 3
     }
 
-    public static class RoleStrings {
-        public static string GetRoleDescription(Role role) { return GetRoleDescription(role.ToString()); }
+    public static class RoleStrings
+    {
+        public static string GetRoleDescription(Role role)
+        {
+            return GetRoleDescription(role.ToString());
+        }
+
         public static string GetRoleDescription(string role)
         {
-            switch(role)
+            switch (role)
             {
                 case "Owner": return "Accept agreements, view information and manage PAYE schemes, organisations, apprentices and team members";
                 case "Transactor": return "Add apprentices and view information";
                 case "Viewer": return "View information but can’t make changes";
                 default: throw new ArgumentException("Unexpected role: " + role);
             }
         }
-        
-        public static string GetRoleDescriptionToLower(Role role) { return GetRoleDescriptionToLower(role.ToString()); }
```

Comment:
Are these methods required on this side? they don't seem to be used. See we have the same class in EmployerAccounts that we do here, so they don't share the Enum, don't have a shared library so guess it's a necessary evil?

Reply:
That class has been deleted.